### DISCUSSION TITLE
0.6r1290&r1318 has some BUGS 是不是很多BUG问题?

### DISCUSSION THREAD

**Discussion by Fanturner** [emukidid/swiss-gc#679] [Q&A]
首先感谢SWISS团队和gc loader团队为NGC玩家持久的做的贡献，谢谢。
说下我遇到的问题：
NGC PAL DOL-001
gc loader 2.0.0.BETA
当使用swiss 0.6r1290&r1318两个版本的时候，玩PhantasyStarOnline1+2PLUS USA时候，会有很多问题，打不破箱子，穿模箱子，有时候打不到怪物，有时候捡不起掉落物品，有时候突然黑屏机器重启。经过测试这些问题在r1250都不存在。
为什么呢？是我没有设置好哪里么？

最后再次感谢无私的SWISS团队。

First，thank to the SWISS team and the gc loader team for your lasting contributions to NGC players. Thank you.

Let me tell you the problem I encountered:
NGC PAL DOL-001
gc-loader 2.0.0.BETA
When using the two versions of swiss 0.6r1290&r1318,  playing PhantasyStarOnline1+2PLUS USA, there will be many problems, such as can't break the box, pass through the box, sometimes can't beat monsters, sometimes can't pick up dropped items, sometimes suddenly Black screen machine restarts. these problems do not exist in the r1250.
why? Am I not setting right?

Finally, thanks again to the selfless SWISS team for gamecube gamer.

**Comment by Extrems** [comment]
See issue #673. They should be fixed with the [latest development build](https://github.com/emukidid/swiss-gc/suites/6582664714/artifacts/246727736).

**Reply by Fanturner** [reply]
> See issue #673. They should be fixed with the [latest development build](https://github.com/emukidid/swiss-gc/suites/6582664714/artifacts/246727736).

ok ,THX a lot!

**Reply by Fanturner** [reply]
> See issue #673. They should be fixed with the [latest development build](https://github.com/emukidid/swiss-gc/suites/6582664714/artifacts/246727736).

刚才森林打了一趟龙，r1329基本正常了，但还是有些地上的物品捡不起来。
I fought a dragon boss in the forest just now, r1329 is basically ok, but there are still some items on the ground that I can't pick up. TAT
